atomic_reactor/utils/cachito.py

```diff
@@ -101,7 +102,7 @@ def request_sources(self, repo, ref, flags=None, pkg_managers=None, user=None,
         return response_json
 
     def wait_for_request(
-            self, request, burst_retry=1, burst_length=30, slow_retry=10):
+            self, request, burst_retry=5, burst_length=30, slow_retry=10):
```

Comment:
Why is this also being changed? Should this change be in a different PR or also be explained in the commit message?

Reply:
I added an explanation in the commit message and PR description. I also changed this to `3` seconds.